src/main/java/org/primefaces/component/commandlink/CommandLinkRenderer.java

```diff
@@ -90,12 +90,7 @@ public void encodeEnd(FacesContext context, UIComponent component) throws IOExce
             writer.writeAttribute("id", clientId, "id");
             writer.writeAttribute("href", "#", null);
             writer.writeAttribute("class", styleClass, null);
-            if (link.getAriaLabel() != null) {
-                writer.writeAttribute(HTML.ARIA_LABEL, link.getAriaLabel(), null);
-            }
-            else if (link.getTitle() != null) {
-                writer.writeAttribute(HTML.ARIA_LABEL, link.getTitle(), null);
-            }
+            writer.writeAttribute(HTML.ARIA_LABEL, link.getAriaLabel(), null);
```

Comment:
Is this right?  Shouldn't it be...
```java
if (link.getAriaLabel() != null) {
   writer.writeAttribute(HTML.ARIA_LABEL, link.getAriaLabel(), null);
}
```

Reply:
At least the HtmlResponseWriter does a not null check. 
https://github.com/eclipse-ee4j/mojarra/blob/e85dcd755bec27008dc70446b56bd30d148b2f2b/impl/src/main/java/com/sun/faces/renderkit/html_basic/HtmlResponseWriter.java#L712-L714

And in CommandButtonRenderer there is no extra null check.
https://github.com/primefaces/primefaces/blob/c49696d66ff3216a849a6d81ccd821f553bcda2c/src/main/java/org/primefaces/component/commandbutton/CommandButtonRenderer.java#L86